index.html

```diff
@@ -0,0 +1,604 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>TNW</title>
+</head>
+
+<body>
+        <section class="response-logo">
+            <img class="res-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="reponse-logo">
+        </section>
+        <nav class="sm-nav">
+            <a href="#"><i class="fas fa-bars" id="menu"></i></a>
+            <a href="#"><i class="fas fa-search" id="sm-search"></i></a>
+        </nav>
+    <header class="nav-container">
+        <nav>
+        <div class="tnw-gif">
+        <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="TNW">
+        </div>
+        <section class="nav-sub-container">
+        <div class="navigation1">
```

Comment:
Your design is having double vertical scroll bars, let's get rid of the inner one. 

![vert](https://user-images.githubusercontent.com/31304943/69760025-bf787b00-1163-11ea-8b21-92a38d1be239.png)